cloudflare/schema_cloudflare_teams_proxy_endpoints.go

```diff
@@ -0,0 +1,26 @@
+package cloudflare
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func resourceCloudflareTeamsProxyEndpointSchema() map[string]*schema.Schema {
+
+	return map[string]*schema.Schema{
+		"account_id": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"name": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"subdomain": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"ips": {
+			Type:     schema.TypeList,
```

Comment:
do you care about order? if not, you probably want `TypeSet`.

Reply:
nice, done